.gitlab/llnl-ci-spack.yml

```diff
@@ -0,0 +1,175 @@
+variables:
+  SPACK_WORK_DIR: $SDK_WORK_DIR/spack
+  RADICAL_PILOT_DBURL: ${MONGODB_CONNECTION_STRING}?tlsAllowInvalidCertificates=true
+  TEST_TYPE: "spack"
+
+stages:
+  - setup
+  - test
+  - cleanup
+
+.final_steps:
+  script: &finalize
+    - chgrp -fR exaworks $SPACK_WORK_DIR || true
+    - chmod -fR 02770 $SPACK_WORK_DIR    || true
+
+spack-setup:
+  stage: setup
+  tags:
+    - quartz
+    - shell
+  script:
+    - mkdir -p $SPACK_WORK_DIR
+    - test -d $SPACK_WORK_DIR/spack && exit 0
```

Comment:
will be only one line -> `test -d $SPACK_WORK_DIR && exit 0`

Reply:
Done